javascript-carousel/main.js

```diff
@@ -0,0 +1,76 @@
+var $next = document.querySelector('.next');
+var $previous = document.querySelector('.previous');
+var $currentImage = document.querySelector('.first');
+var $radio1 = document.querySelector('.dot1');
+var $radio2 = document.querySelector('.dot2');
+var $radio3 = document.querySelector('.dot3');
+var $radio4 = document.querySelector('.dot4');
+var $radio5 = document.querySelector('.dot5');
+
+var clickCounter = 0;
+var clickBack = 0;
+
+$next.addEventListener('click', function (event) {
+  clickCounter++;
+  if (clickCounter > 0 && clickCounter <= 1) {
```

Comment:
Can this just be `if(clickCounter === 1)`? I believe it can, and that is less intensive and easier to read. You should adjust the format of each of the other conditions in this conditional block as well.